docs/devices/FlashingCertificate/CLI.rst

```diff
@@ -29,13 +29,13 @@ Provisioning of the certificate using CLI also results in the following actions:
 
 .. only:: v1_5_x
 
-    1. Programming of the `AT host sample <https://github.com/NordicSemiconductor/at_host-hex>`_ onto the device.
+    1. Programming of the `AT Client sample <https://github.com/NordicSemiconductor/at_host-hex>`_ onto the device.
```

Comment:
The readme.md in the link target still says AT Host, which needs to be changed as well.

Reply:
I disagree. "AT Client" is the wrong name for the sample (however it is kept for historical reasons in NCS). The sample runs an AT command host, not a client, therefore the linked project has the correct name.

If link must match the title:

```suggestion
    1. Programming of the `AT Client sample <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf9160/at_client/README.html>`_ onto the device.
```

Reply:
Changed the link to point to the NCS sample.